scripts/dist.sh

```diff
@@ -8,6 +8,38 @@ if [ -z "${VERSION}" ]; then
   exit 1
 fi
 
+gpg_signing_key=348FFC4C
+
+verify_hc_releases() {
+    if ! command -v  hc-releases 2>/dev/null >/dev/null
+    then
+        echo "hc-releases binary is not present" >&2
+        exit 1
+    fi
+}
+
+verify_gpg_key() {
+    if ! gpg --list-keys "${gpg_signing_key}" >/dev/null 2>/dev/null
+    then
+        echo "gpg key ${gpg_signing_key} is not present" >&2
+        exit 1
+    fi
+}
+
+verify_s3_access() {
+    if ! aws s3 ls s3://hc-releases/ >dev/null 2>/dev/null
```

Comment:
Just realized this won't work without a profile for me.

Reply:
I'll follow up with using `AWS_PROFILE` variable, to ensure that this aws command uses the same profile as `hc-releases` command below.